apps/member-profile/app/routes/_profile.companies_.$id.tsx

```diff
@@ -156,9 +184,24 @@ export default function CompanyPage() {
       </header>
 
       <Text color="gray-500">{company.description}</Text>
-      <ReviewsList />
-      <CurrentEmployees />
-      <PastEmployees />
+      <nav>
+        <ul className="flex flex-wrap gap-x-4 gap-y-2">
+          <NavigationItem to="?tab=employees">Employees</NavigationItem>
+          <NavigationItem to="?tab=reviews">Company Reviews</NavigationItem>
+          <NavigationItem to="?tab=interview-reviews">
+            Interview Reviews
+          </NavigationItem>
+        </ul>
+      </nav>
+      <Divider my="4" />
+      {activeTab === 'employees' && (
+        <>
+          <CurrentEmployees />
+          <PastEmployees />
```

Comment:
Might make more sense to have `CurrentEmployees` and `PastEmployees` under a single component named `Employees`